### PR TITLE
fix migration 3dda56f1c

### PR DESCRIPTION
**Pull request by timifasubaa**
This PR fixes the issue with slices that have no params and pass None to be decoded by json.loads.

@betodealmeida @graceguo-supercat @mistercrunch  @john-bodley

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@ geopy==1.11.0
 gunicorn==19.8.0
 humanize==0.5.1
 idna==2.6
+isodate==0.6.0
```
Comment (john-bodley) [inline requirements.txt]: Why do we need `isodate`?

Comment (mistercrunch) [inline requirements.txt]: unused

File: setup.py [hunk 2]
```diff
@@ -75,6 +75,7 @@ def get_git_sha():
         'gunicorn',  # deprecated
         'humanize',
         'idna',
+        'isodate',
```
Comment (mistercrunch) [inline setup.py]: unused

**Review by john-bodley — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (timifasubaa) [reply setup.py]: Migration line 109

**Pull request merged**